src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -411,6 +411,18 @@ private function addHttpCacheSection(ArrayNodeDefinition $rootNode): void
                                     ->end()
                                     ->info('To pass options to the client charged with the request.')
                                 ->end()
+                                ->booleanNode('xkey')
+                                    ->defaultFalse()
+                                    ->info('Enable support for xkey invalidation')
+                                ->end()
+                                ->scalarNode('xkey_glue')
```

Comment:
IMO it would be better to have a `xkey` config attribute, with an `enabled` and `glue` sub-attributes.

Reply:
this is part of the `invalidation` we want to support both as the servers on the invalidation node are used there as well.

Reply:
Not sure I understand what you mean. We are in the `invalidation` part, and there are two modes: `xkey` and `http_tags`.
Each one should have its own configuration.

Reply:
does this work with `canBeEnabled`?
```
xkey: false
```

or we need the full 

```
xkey:
  enabled: true
```

Reply:
Yes! https://github.com/symfony/symfony/blob/eb2cae6b4901b2d5ab7d394426c262e4c2c3a4a6/src/Symfony/Component/Config/Definition/Builder/ArrayNodeDefinition.php#L304